test/parallel/test-transform-by.js

```diff
@@ -286,17 +283,19 @@ async function transformByThrowPriorToForAwait() {
     strictEqual(err.message, 'kaboom');
   }));
 
-  read.pipe(stream);
+  read.pipe(stream).resume();
 }
 
 Promise.all([
   transformBy(),
   transformByFuncReturnsObjectWithSymbolAsyncIterator(),
-  transformByObjReturnedWSymbolAsyncIteratorWithNonPromiseReturningNext(),
-  transformByObjReturnedWSymbolAsyncIteratorWithNoNext(),
-  transformByObjReturnedWSymbolAsyncIteratorThatIsNotFunction(),
+  // NOTE: These should be handled by Readable.from.
+  // transformByObjReturnedWSymbolAsyncIteratorWithNonPromiseReturningNext(),
+  // transformByObjReturnedWSymbolAsyncIteratorWithNoNext(),
+  // transformByObjReturnedWSymbolAsyncIteratorThatIsNotFunction(),
   transformByFuncReturnsObjectWithoutSymbolAsyncIterator(),
-  transformByEncoding(),
+  // NOTE: This doesn't make sense for iterable? Is it consistent with Readable.from?
+  // transformByEncoding(),
```

Comment:
I'm not sure what this test, tests for?

Reply:
in order to have an analogue to `(chunk, enc, cb) => { }` we add an `encoding` property for cases where transform streams behave differently depending on the encoding.

Reply:
I don't think this is taken into account in `Readable.from` and since we are doing `objectMode` I'm still unsure what the semantics here should be. I need to think about this a bit more unless someone can clarify?